src/cli_plugin/install/downloaders/http.js

```diff
@@ -2,11 +2,47 @@ import Wreck from 'wreck';
 import Progress from '../progress';
 import { fromNode as fn } from 'bluebird';
 import { createWriteStream } from 'fs';
+import HttpProxyAgent from 'http-proxy-agent';
+import URL from 'url';
 
-function sendRequest({ sourceUrl, timeout }) {
+function getProxyAgent(sourceUrl, logger) {
+  const httpProxy = process.env.http_proxy || process.env.HTTP_PROXY;
+  // we have a proxy detected, lets use it
+  if (httpProxy) {
```

Comment:
Do we have some styleguide about early return? I am personally not a fan of early return, if it's not the very first statement and just checks function parameters.